### PR TITLE
JNLP examples first version

### PR DESCRIPTION
**Pull request by GabrielaReyna**
Examples for application and applet

### CODE REVIEW

**Comment by GabrielaReyna** [comment]
I think the links in the code blocks are being checked

File: src/content/docs/cheerpj3/13-examples/02-jnlp.mdx [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+    <j2se version="1.4+" initial-heap-size="30m" max-heap-size="300m" />
+    <jar href="Pitch.jar"/>
+  </resources>
+  <applet-desc main-class="Pitchview" width="300" height="500"/>
+</jnlp>
+```
+
+### 3. Download the applet file
+
+You will download the applet JAR files by manually building the full URL and pasting it in the browser navigation bar. This is done by appending the `codebase` URL and the `jar` URL.
```
Comment (nanaian) [inline src/content/docs/cheerpj3/13-examples/02-jnlp.mdx:229]:
```suggestion
Download the applet JAR files by manually building the full URL and pasting it in the browser navigation bar. This is done by concatenating the `codebase` URL and the `jar` URL.
```

File: src/content/docs/cheerpj3/13-examples/02-jnlp.mdx [hunk 2]
```diff
@@ -0,0 +1,363 @@
...
+For example:
+
+```
+https://raw.githubusercontent.com/leaningtech/cheerpj-example-applet/main/Pitch.jar
+
+```
+
+### 3. Create a project directory
+
+Create a directory where all the files will live. You can choose any name such `cheerpj-example-applet`:
```
Comment (nanaian) [inline src/content/docs/cheerpj3/13-examples/02-jnlp.mdx:239]:
```suggestion
Create a directory where all the files will live. You can choose any name, such as `cheerpj-example-applet`:
```

**Review by nanaian — CHANGES_REQUESTED** [review]

**Comment by nanaian** [comment]
> I think the links in the code blocks are being checked

Modify the `exclude` regexes in `lychee.toml` to resolve this

**Pull request merged**